task4.py

```diff
@@ -0,0 +1,18 @@
+""""
+Задача 8: Требуется определить, можно ли от шоколадки размером n × m долек отломить k долек, если разрешается сделать один разлом по прямой между дольками (то есть разломить шоколадку на два прямоугольника).
+
+*Пример:*
+
+3 2 4 -> yes
+3 2 1 -> no
+"""
+
+
+n = int(input("Ширина: "))
+m = int(input("Длина: "))
+k = int(input("На сколько долек ломаем?"))
+
+if k < n * m and ((k % n == 0) or (k % m == 0)):
```

Comment:
Первое условие можно расширить относительно нижней границы допустимых значений переменной `k`. А в правой скобке внутренние скобки не нужны.
```suggestion
if 0 < k < n * m and (k % n == 0 or k % m == 0):
```